srcs/client/src/App.tsx

```diff
@@ -38,29 +58,28 @@ function App() {
       contextValue.updateIsConnected(true);
     } else contextValue.updateIsConnected(false);
   });
-
   /*
    ** Context.Provider surround all routes and spread the contextValue, BrowserRouter allows us to use routes.
    ** Routes surround all route
    */
   return (
     <Context.Provider value={contextValue}>
       <BrowserRouter>
-          <NavBar />
+          <NavBar userID={userID}/>
```

Comment:
Idea: we could add userID to the context so any component can use it use a useContext

Reply:
Yep it would make our code less verbose especially when using useNavigate()